src/subscript/field_statistics/field_statistics.py

```diff
@@ -608,17 +612,20 @@ def read_ensemble_realization(
     iter_number,
     property_param_name,
     zone_code_names,
+    geogrid_name,
 ):
     realization_path = Path(f"realization-{realization_number}/iter-{iter_number}")
-    grid_path = Path("share/results/grids/geogrid.roff")
+    grid_path = Path("share/results/grids/" + geogrid_name.strip() + ".roff")
```

Comment:
I think you don't need to strip `geogrid_name` anymore since you already do it when you read it from yaml file?

Reply:
fixed